dataswarm/manager/dataswarm_manager.c

```diff
@@ -119,7 +120,21 @@ void handle_client_message( struct dataswarm_manager *m, struct dataswarm_client
 	if(!strcmp(action,"task_submit")) {
 		/* */
 	} else if(!strcmp(action,"file_submit")) {
-		/* */
+        // read in json message
+        const char *params = jx_lookup_string(msg, "params");
+	   
+        // convert params argument to jx
+        struct jx *file = jx_parse_string(params);
+        
+        // declare_file()
+        int uuid = dataswarm_declare_file(file);
+
+        /* create jx of uuid response */
+        struct jx *response;
```

Comment:
Need to create a JX object and add something to it.  (e.g. the response and the UUID.)